src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.h

```diff
@@ -109,7 +115,7 @@ const AttestationTrustStore * GetTestAttestationTrustStore();
  *          process lifetime.  In particular, after the first call it's not
  *          possible to change which AttestationTrustStore is used by this verifier.
  */
-DeviceAttestationVerifier * GetDefaultDACVerifier(const AttestationTrustStore * paaRootStore);
+DeviceAttestationVerifier * GetDefaultDACVerifier(const AttestationTrustStore * paaRootStore, const RevocationSet * revocationSet);
```

Comment:
So this requires the revocation set to be available up front, right?

Reply:
Yes, but due to the CI failing we are changing this to a default argument nullptr.

Reply:
Per the current design, RevocationSet will be created from DCL first and a JSON file with the list of revocations will be generated. This JSON file is then fed into the app.

Reply:
We will not be using JSON in production. The revocation set accessor interface should do all the necessary async work to determine if a certificate is revoked.

Reply:
This change has been removed from this PR.